kong.conf.default

```diff
@@ -415,26 +415,42 @@
 # they have been set.
 
 #dns_resolver =                  # Comma separated list of nameservers, each
-                                 # entry in `ipv4[:port]` format to be used by
+                                 # entry in `ip[:port]` format to be used by
                                  # Kong. If not specified the nameservers in
                                  # the local `resolv.conf` file will be used.
-                                 # Port defaults to 53 if omitted.
+                                 # Port defaults to 53 if omitted. Accepts
+                                 # both IPv4 and IPv6 addresses.
 
 #dns_hostsfile = /etc/hosts      # The hosts file to use. This file is read
                                  # once and its content is static in memory.
                                  # To read the file again after modifying it,
                                  # Kong must be reloaded.
 
-#dns_order = LAST,SRV,A,CNAME    # the order in which to resolve different
+#dns_order = LAST,SRV,A,CNAME    # The order in which to resolve different
                                  # record types. The `LAST` type means the
                                  # type of the last successful lookup (for the
                                  # specified name). The format is a (case
                                  # insensitive) comma separated list.
 
-#dns_not_found_ttl = 30.0        # ttl in seconds for empty DNS responses and
+#dns_stale_ttl = 4               # Once a record expires, how much longer (in
+                                 # seconds) may the current stale record be 
+                                 # used while the refresh query is executed in
+                                 # the background.
```

Comment:
"Defines, in seconds, how long a record will remain in cache past its TTL. This value will be used while the new DNS record is fetched in the background." ?